CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 * [CHANGED] [5038](https://github.com/stripe/stripe-android/pull/5038) Remove force portrait mode in Google Pay.
 
 ### PaymentSheet
-- [FIXED] the format of the country dropdown in PaymentSheet for all languages.
```

Comment:
Accidental?

Reply:
This was originally based off of another branch that has since been merged, so it set the new base to master, but then there were merge conflicts.  They are now resolved.